### PR TITLE
Implementation of a Skywalker output file diff tool.

### PR DESCRIPTION
**Pull request by overfelt**
Fixes #1

### CODE REVIEW

File: tools/skywalker_diff.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# This script generates norms for cross-validating the particle nucleation
+# process. To generate the data needed by the script, run skywalker in the
+# current directory, using nucleation.yaml as input. This writes a file called
+# haero_skywalker.py to the directory, which is then imported as a module.
+# Have another haero_skywalker.py created previously.  This script
```
Comment (jeff-cohere) [inline tools/skywalker_diff.py]: This implies that both modules would be named `haero_skywalker`, right? Maybe we want to talk about `mam_skywalker` or `boxmodel_skywalker` vs `haero_skywalker` instead?

File: tools/skywalker_diff.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    keys2 = list(vars2)
+    keys = [key for key in keys1 if key in keys2]
+    for key in keys :
+        vals1 = np.array(vars1[key])
+        vals2 = np.array(vars2[key])
+        (vals1, vals2) = check_lens(f, key, vals1, vals2)
+        # check_vals is either the check_input or check_output function
+        check_vals(f, key, vals1, vals2)
+          
+def parse_args () :
```
Comment (jeff-cohere) [inline tools/skywalker_diff.py:59]: 👍

**Review by jeff-cohere — APPROVED** [review]
Looks good! I left a minor comment about one of your comments. :-)

File: tools/skywalker_diff.py (hunk 1 above)
Comment (overfelt) [reply tools/skywalker_diff.py]: Yes, if someone wants to compare the Skywalker results over time or between MAMRefactor and Hearo the results files will need to be organized to have different names or different directories.   I think this is the standard problem of organizing output files for comparison.  I tried to clarify by updating the comment.

**Comment by jeff-cohere** [comment]
Good to go! Do you have merge perms on this repo?

**Comment by overfelt** [comment]
I'm about to find out if I have merge permissions.....

**Comment by overfelt** [comment]
Yep, seems I do.

**Pull request merged**